src/electron/lanceDB/lancedb.ts

```
@@ -0,0 +1,170 @@
+import { connect } from "@lancedb/lancedb";
+
+interface ImageEmbedding {
+  id: number;
+  filename: string;
+  description: string;
+  vector: Float32Array;
+}
+
+export class LanceDBManager {
+  private static instance: LanceDBManager;
+  private db: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)